.github/workflows/render-rmds.yml

```diff
@@ -7,7 +7,7 @@ on:
     paths: 
       - '**.Rmd'
       - '!**-live.Rmd' # don't trigger for live-only changes
-      - '!**exercise*.Rmd' # or exercise notebooks
+      - '!**/exercise*.Rmd' # or exercise notebooks
```

Comment:
If I am not mistaken, this `!**/exercise` syntax is excluding the exercise notebook files anywhere in the repo (well according to the [GHA workflow syntax docs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions)). If that's the case, is there a reason why this syntax is not propagated to the `-live` and `.Rmd` files?

Reply:
The `**` already matches any path, including any `/`s. The reason I include the `/` in the exercise line is so that "exercise" has to be the first part of the file name. This is so we don't accidentally match something like "05-exercised_mouse_expression.Rmd" if it isn't actually an "exercise" notebook.

Reply:
Ah gotcha!! 👍